src/main/java/org/truffleruby/core/TruffleSystemNodes.java

```diff
@@ -273,4 +275,19 @@ protected int availableProcessors() {
 
     }
 
+    @CoreMethod(names = "allocated_bytes_of_current_thread", onSingleton = true)
+    public abstract static class AllocatedBytesNode extends CoreMethodArrayArgumentsNode {
+        private static ThreadMXBean bean;
+
+        @TruffleBoundary
+        @Specialization
+        public static long allocatedBytes() {
```

Comment:
should be `protected` as the CI says

Reply:
Fixed. Too much Operations DSL... There the standard is `public static`.